tracer/src/Datadog.Trace/Agent/StatsAggregationKey.cs

```diff
@@ -14,19 +14,22 @@ namespace Datadog.Trace.Agent
         public readonly string OperationName;
         public readonly string Type;
         public readonly int HttpStatusCode;
+        public readonly bool Synthetics;
 
         public StatsAggregationKey(
             string resource,
             string service,
             string operationName,
             string type,
-            int httpStatusCode)
+            int httpStatusCode,
+            bool synthetics)
         {
             Resource = resource;
             Service = service;
             OperationName = operationName;
             Type = type;
             HttpStatusCode = httpStatusCode;
+            Synthetics = synthetics;
```

Comment:
Nit-  we're talking of a span generated from a Synthetics call, right? Should we rename this one to `isSyntheticsCall`

Reply:
Good idea, renamed in 403bcbb30ea3afd9c4ecbbf6057781de53ff5e3a and 9ac7f62f002f1c974d3634aaba0fcbfb1b112368